servo-build-dependencies/android.sls

```diff
@@ -28,6 +28,7 @@ android-dependencies:
       - lib32z1
       - libstdc++6
       - libgl1-mesa-dev
+      - unzip
```

Comment:
It looks like Salt sometimes uses the binary and sometimes Python's zipfile module, so let's keep this even with `archive.extracted`.